packages/datadog-instrumentations/src/redis.js

```diff
@@ -11,6 +11,8 @@ const startCh = channel('apm:redis:command:start')
 const finishCh = channel('apm:redis:command:finish')
 const errorCh = channel('apm:redis:command:error')
 
+const state = []
```

Comment:
Rather than all this `state.url` stuff I would just change this to a `let url` and assign directly to that. At the least though this should be an object, not an array. Using an array as an object like this could cause a deopt.

Reply:
Also, looks like this should be an object not an array